Sources/x10/xla_tensor/ops/as_strided_view_update.cpp

```diff
@@ -47,7 +47,7 @@ xla::XlaOp LowerAsStridedViewUpdate(xla::XlaOp target, xla::XlaOp input,
     xla::XlaOp r1_target = XlaHelpers::Flatten(target);
     transposed_input = xla::DynamicUpdateSlice(
         r1_target, r1_input,
-        {XlaHelpers::ScalarValue<xla::int64>(storage_offset, input.builder())});
+        absl::Span<const xla::XlaOp>({XlaHelpers::ScalarValue<xla::int64>(storage_offset, input.builder())}));
```

Comment:
Out of curiosity, why do we need this change (and the other one to the header), isn't this going to use the same compiler we use for Kokoro?

Reply:
This is because it is using a different compiler in the outside VM. I think I'm going to try and add my caching to the other bazel build command.